### PR TITLE
Feature/import from google play

### PR DESCRIPTION
**Pull request by greymag**
L10n: Command `import_xml` to import translations from Google Play to the projects's xml files.

### CODE REVIEW

File: lib/commands/l10n/import_xml_command.dart [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      : super(
+          'import_xml',
+          'Import transalations from Google Play '
+              "to the project's xml files",
+        ) {
+    argParser
+      ..addOption(
+        _argPath,
+        abbr: 'p',
+        help: 'Path to the durectory with translations from Google Play.',
```
Comment (ChessMax) [inline lib/commands/l10n/import_xml_command.dart:23]:
```suggestion
        help: 'Path to the directory with translations from Google Play.',
```

File: lib/commands/l10n/import_xml_command.dart [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+          final googlePlayLocale = name.replaceFirst('${translationUid}_', '');
+          final locale = _convertGooglePlayLocale(googlePlayLocale);
+
+          final subdirName = '${translationUid}_${googlePlayLocale}';
+          final sourceFile = await _requireFile(path.join(
+              sourceDir.path, subdirName, '${subdirName}_${projectUid}.xml'));
+          final targetFile = await _requireFile(
+              path.join(config.getXmlFilesPath(locale), filename));
+
+          printVerbose(' Copy $sourceFile to $targetFile');
```
Comment (ChessMax) [inline lib/commands/l10n/import_xml_command.dart:74]:
```suggestion
          printVerbose('Copy $sourceFile to $targetFile');
```

File: lib/commands/l10n/import_xml_command.dart [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+          await sourceFile.copy(targetFile.path);
+
+          imported.add(locale);
+        }
+      }
+    }
+
+    return success(
+        message: 'Success. Imported locales (${imported.length}): '
+            '${imported.join(", ")}.');
```
Comment (ChessMax) [inline lib/commands/l10n/import_xml_command.dart:84]: We don't use a method invocation inside string interpolation, do we?

File: lib/commands/l10n/import_xml_command.dart [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  }
+
+  Future<Directory> _requireDirectory(String path) async {
```
Comment (ChessMax) [inline lib/commands/l10n/import_xml_command.dart:87]: Shouldn't it be somewhere in `file_utils` or something? Maybe a todo should be added at least.

File: lib/commands/l10n/import_xml_command.dart [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    final dir = Directory(path);
+    final exist = await dir.exists();
+    if (!exist) throw RunException(1, 'Directory $path is not exist');
+    return dir;
+  }
+
+  Future<File> _requireFile(String path) async {
```
Comment (ChessMax) [inline lib/commands/l10n/import_xml_command.dart:94]: Shouldn't it be somewhere in `file_utils` or something? Maybe a todo should be added at least.

File: README.md [hunk 6]
```diff
@@ -112,6 +112,17 @@ And to the iOS localization:
 alex l10n from_xml --to=ios
 ```
 
+#### Import translation from Google Play to project XML files
+
+When you download and unzip translations from Google Play,
+you need to import it in project's xml files. You can
```
Comment (ChessMax) [inline README.md:118]:
```suggestion
you need to import them in project's xml files. You can 
```

File: README.md [hunk 7]
```diff
@@ -112,6 +112,17 @@ And to the iOS localization:
 alex l10n from_xml --to=ios
 ```
 
+#### Import translation from Google Play to project XML files
+
+When you download and unzip translations from Google Play,
+you need to import it in project's xml files. You can 
+copy it all manually, but it's very inconvenient.
+So you can use command `import_xml` to do it.
```
Comment (ChessMax) [inline README.md:120]:
```suggestion
So you can use the command `import_xml` to do it.
```

File: lib/commands/l10n/import_xml_command.dart [hunk 8]
```diff
@@ -0,0 +1,104 @@
+import 'dart:io';
+
+import 'package:alex/alex.dart';
+import 'package:alex/commands/l10n/src/l10n_command_base.dart';
+import 'package:alex/src/exception/run_exception.dart';
+import 'package:path/path.dart' as path;
+
+/// Command to import transalations from Google Play
```
Comment (ChessMax) [inline lib/commands/l10n/import_xml_command.dart:8]:
```suggestion
/// Command to import translations from Google Play
```

File: lib/commands/l10n/import_xml_command.dart [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+/// to the project's xml files.
+class ImportXmlCommand extends L10nCommandBase {
+  static const _argPath = 'path';
+
+  ImportXmlCommand()
+      : super(
+          'import_xml',
+          'Import transalations from Google Play '
```
Comment (ChessMax) [inline lib/commands/l10n/import_xml_command.dart:16]:
```suggestion
          'Import translations from Google Play '
```

File: lib/commands/l10n/import_xml_command.dart [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+              "to the project's xml files",
```
Comment (ChessMax) [inline lib/commands/l10n/import_xml_command.dart:17]:
```suggestion
              'to the project's xml files',
```

File: lib/commands/l10n/import_xml_command.dart (hunk 3 above)
Comment (greymag) [reply lib/commands/l10n/import_xml_command.dart:84]: Thought to make an exception) Ok, i'll fix it

File: lib/commands/l10n/import_xml_command.dart (hunk 10 above)
Comment (greymag) [reply lib/commands/l10n/import_xml_command.dart:17]: No, it's can't be done - because of "project's"

**Pull request merged**